nest-tax-backend/src/utils/middlewares/logger.ts

```diff
@@ -19,34 +23,47 @@ export class AppLoggerMiddleware implements NestMiddleware {
           ?.toString()
           .includes('application/json')
       ) {
-        responseData = exitData.toString().slice(0, 1000)
+        // Filter out keys starting with `$`. We will log them later
+        const { responseLog, responseMessage } = separateLogFromResponseObj(
+          typeof exitData === 'string' ? JSON.parse(exitData) : exitData,
+        )
+        // eslint-disable-next-line no-param-reassign
+        exitData = JSON.stringify(responseMessage)
+
+        responseData = exitData.toString().slice(0, 2000)
```

Comment:
Why this arbitrary value of 2000?

Reply:
Originally, the value was set to 1000, but I noticed that this was causing some logs I wanted to review to be cut.